CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.4.1 2016-05-1
```

Comment:
I appreciate that you're filling in the CHANGELOG, but this version and release date are wrong. Also, there are other features that will be in the next request. Will you please remove this? I'll attribute you when I write the notes before the release.